apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -77,6 +77,17 @@ const buildColorExample = token => {
     return `<div class="${classes.join(' ')}" style="${cssVariable}";></div>`;
 };
 
+/**
+ * Builds the example spacing swatch to show on the token list item
+ * @param {string} token - the token value i.e. 24, 80
+ * @returns {string} - the spacing swatch example HTML string
+ */
+const buildSpacingExample = token => {
+    const cssVariable = `--example-spacing: ${token}px`;
+
+    return `<div class="c-tokensTable-example-spacing-container"><div class="c-tokensTable-example--spacing" style="${cssVariable}";></div></div>`;
```

Comment:
This semicolon looks a bit out of place to me
```suggestion
    return `<div class="c-tokensTable-example-spacing-container"><div class="c-tokensTable-example--spacing" style="${cssVariable}"></div></div>`;
```